src/components/ThumbnailImage.js

```diff
@@ -54,7 +62,11 @@ class ThumbnailImage extends PureComponent {
         } else {
             thumbnailScreenHeight = Math.round(thumbnailScreenWidth * aspectRatio);
         }
+        return {width: thumbnailScreenWidth, height: thumbnailScreenHeight};
+    }
 
+    updateImageSize({width, height}) {
+        const {width: thumbnailScreenWidth, height: thumbnailScreenHeight} = this.calculateThumbnailImageSize(width, height);
         this.setState({thumbnailWidth: thumbnailScreenWidth, thumbnailHeight: Math.max(40, thumbnailScreenHeight)});
```

Comment:
```suggestion
        const {width: thumbnailWidth, height: thumbnailScreenHeight} = this.calculateThumbnailImageSize(width, height);
        this.setState({thumbnailScreenWidth, thumbnailHeight: Math.max(40, thumbnailScreenHeight)});
```

Since `thumbnailScreenHeight` not used other than setting state, it could be named as `thumbnailWidth`

Reply:
Fixed.